src/styles/annotator/bucket-bar.scss

```diff
@@ -35,6 +35,7 @@
     -webkit-tap-highlight-color: rgba(255, 255, 255, 0);
     text-align: center;
     cursor: pointer;
+    margin-top: -8px;
```

Comment:
This entire SASS module needs a little TLC, but for now: there's no need to make `BucketBar` handle top margins.

Reply:
Can you expand on this?

Reply:
See updated comments. The whole approach to styling needs an overhaul; it's in my TODO list.